apps/coscrad-frontend/src/utils/generic-components/style-components/coscrad-main-content-container.tsx

```diff
@@ -12,7 +12,7 @@ export const CoscradMainContentContainer = ({
     return (
         <Container
             component="div"
-            sx={{ paddingTop: '5%', width: { xs: '90%', sm: '80%' }, margin: 'auto' }}
+            sx={{ paddingTop: '5%', width: { xs: '100%', sm: '95%' }, margin: 'auto' }}
```

Comment:
Again, these seem like random little changes that make me nervous given that this is wrapping almost every important UX component. Is there some kind of Cypress test that can guide these changes? Maybe content is running off the page and you can formalize that as a test?